packages/collection-model/lib/model.js

```diff
@@ -232,18 +242,28 @@ const CollectionModel = AmpersandModel.extend(debounceActions(['fetch']), {
   },
 
   /**
-   * @param {{ dataService: import('mongodb-data-service').DataService }} dataService
+   * @param {{
+   *    dataService: import('mongodb-data-service').DataService,
+   *    fetchInfo: boolean,
+   *    force: boolean
+   * }} options
    * @returns
    */
   async fetch({ dataService, fetchInfo = true, force = false }) {
     if (!shouldFetch(this.status, force)) {
       return;
     }
+
+    const instanceModel = getParentByType(this, 'Instance');
+    const { enableDbAndCollStats } = instanceModel.preferences.getPreferences();
```

Comment:
Instead of exposing all the preferences directly from the instance model, can we limit the exposed interface to only what we need here? Something like:

```suggestion
    const enableDbAndCollStats = instanceModel.shouldAllowStatsFetching();
```

As a rule of thumb I'd suggest to try to stay minimal when introducing these sort of cross boudary interfaces so that the intent of the exposed interface is clearer from just the name and the actual implementation can be changed easier

Reply:
that's a good idea, I'll do that!

Reply:
so I ended up using a property instead of a function, as I wanted to make sure we listen to updates on the preferences.